pkgs/top-level/all-packages.nix

```diff
@@ -7867,7 +7867,7 @@ in
 
   tautulli = python3Packages.callPackage ../servers/tautulli { };
 
-  pleroma-otp = callPackage ../servers/pleroma-otp { };
+  pleroma = callPackage ../servers/pleroma { };
```

Comment:
We should add an entry to aliases.nix

Reply:
It's not really the same package and I doubt anyone used the package directly, but sure, adding it won't hurt either

Reply:
done